openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -17,9 +21,6 @@
     deserialize_bytes_to_event_data,
     serialize_event_data_to_bytes,
 )
-# Each new folder with signals must be manually imported in order for the signals to be cached
-#   and used in the unit tests. Using 'disable=reimported' with pylint will work,
-#   because we just use the cached signal list.
 # TODO (EventBus): Find a way to auto-discover these signals.
```

Comment:
[suggestion] Can we just remove the TODO, and fix when and if we decide it is necessary?
```suggestion
```